WooCommerce/WooCommerceTests/System/SessionManagerTests.swift

```diff
@@ -84,50 +84,6 @@ final class SessionManagerTests: XCTestCase {
         XCTAssertEqual(retrieved, Settings.applicationPasswordCredentials)
     }
 
-    /// Verifies that application password is deleted upon calling `deleteApplicationPassword`
-    ///
-    func test_deleteApplicationPassword_deletes_password_from_keychain() {
```

Comment:
These tests are removed as they are incorrect. Unit tests should only test immediate functionalities.